src/lib.rs

```diff
@@ -1153,12 +1155,6 @@ mod panic {
 
 struct LoggerGuard(&'static Log);
 
-impl Drop for LoggerGuard {
```

Comment:
Thanks for the guidance, here!  As I feared, I fundamentally misunderstood the ask.  I'll clean this up and repush shortly.